modules/placement-driver/src/main/java/org/apache/ignite/internal/placementdriver/LeaseUpdater.java

```diff
@@ -447,19 +448,24 @@ private void updateLeaseBatchInternal() {
                 );
             }
 
+            if (Arrays.equals(leasesCurrent.leasesBytes(), renewedValue)) {
+                LOG.info("No leases to update found.");
+                return;
+            }
+
             msManager.invoke(
                     or(notExists(key), value(key).eq(leasesCurrent.leasesBytes())),
                     put(key, renewedValue),
                     noop()
             ).whenComplete((success, e) -> {
                 if (e != null) {
-                    LOG.error("Lease update invocation failed", e);
+                    LOG.error("Lease update invocation failed.", e);
 
                     return;
                 }
 
                 if (!success) {
-                    LOG.debug("Lease update invocation failed");
+                    LOG.warn("Lease update invocation failed.");
```

Comment:
do we need warn here?

Reply:
I think a warning is better than just hiding this issue. In ordinary cases, the PD should process all the updates.
Currently, this message might appear because we have a concurrent update of the MC key.